scripts/jest.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+setup_service google-chrome-stable 83.0.4103.61-1
```

Comment:
Does jest require chrome?

Reply:
Hmm no I don't think it does. Removing this line might speed it up. Let me give it a try.

Reply:
Yep! shaved 3 minutes